src/components/EventList.js

```diff
@@ -29,7 +29,7 @@ export default function(store){
       },
       isSelectedZipcodeInvalid() {
         return this.filters.zipcode &&
-          this.zipcodes &&
```

Comment:
Looks like zipcodes is either an empty object or a filled object, either way the following `Object.keys(this.zipcodes).length` will catch it.